nikola/post.py

```diff
@@ -745,11 +745,16 @@ def section_color(self, lang=None):
 
     def section_link(self, lang=None):
```

Comment:
+1 to just using link. We don't need to have the post access the Nikola object either, tests just need to be resolved after they are written in the document, when copying to output.